src/client/core/utils/replaceCharAt.js

```diff
@@ -0,0 +1,7 @@
+export default function replaceCharAt (str, pos, value) {
```

Comment:
File name should be `replace-char-at.js`.
And you can put it in automation utils because it is used only in automaion. In this case you don't need to export it in testcafeCore